app/uk/gov/hmrc/traderservices/views/UploadFileView.scala.html

```diff
@@ -37,7 +37,7 @@
   backLink = Some(backLink.url)
 ) {
 
-  <form action="@uploadRequest.href" method="POST" enctype="multipart/form-data" class="js-form file-upload"
+  <form action="@uploadRequest.href" method="POST" enctype="multipart/form-data" class="js-form file-upload" novalidate
```

Comment:
Change itself looks good we'll just need to update the error so we get the right message (at present it gives us "The selected file could not be uploaded")

<img width="433" alt="image" src="https://user-images.githubusercontent.com/55190161/100473361-13e76b00-30d6-11eb-8649-114b0bbb72d1.png">